src/dc_mimefactory.rs

```diff
@@ -1193,108 +1111,95 @@ unsafe fn build_body_file(
         } else {
             filename_to_send = dc_get_filename(path_filename)
         }
-        if mimetype.is_null() {
-            if suffix.is_null() {
-                mimetype =
-                    dc_strdup(b"application/octet-stream\x00" as *const u8 as *const libc::c_char)
-            } else if strcmp(suffix, b"png\x00" as *const u8 as *const libc::c_char) == 0 {
-                mimetype = dc_strdup(b"image/png\x00" as *const u8 as *const libc::c_char)
-            } else if strcmp(suffix, b"jpg\x00" as *const u8 as *const libc::c_char) == 0
-                || strcmp(suffix, b"jpeg\x00" as *const u8 as *const libc::c_char) == 0
-                || strcmp(suffix, b"jpe\x00" as *const u8 as *const libc::c_char) == 0
```

Comment:
this variant is not covered by the guess method

Reply:
Added this case into guess method. Or it was deliberate, that `jpe` extension was recognized in one situation, but not in another?

Reply:
not sure, cc @r10s